adapters/http4s/src/test/scala/caliban/Http4sAdapterSpec.scala

```diff
@@ -30,6 +31,8 @@ object Http4sAdapterSpec extends ZIOSpecDefault {
   type Env         = TestService with Uploads
   type TestTask[A] = RIO[Env, A]
 
+  private implicit val network: Network[TestTask] = Network.forAsync[TestTask]
```

Comment:
Am I blind for not seeing this in Release Notes of http4s and fs2? Is it documented anywhere?

Reply:
Hmm yeah, it doesn't seem to be documented anywhere, but the `.default` method on ember builder now requires a `Network[F]` implicit. Not sure how this isn't a breaking change 🤔

Reply:
Weird, it looks like it was done in 0.23.16 https://github.com/http4s/http4s/releases/tag/v0.23.16